security/pfSense-pkg-acme/files/usr/local/pkg/acme/acme_sh.inc

```diff
@@ -254,6 +254,12 @@ EOF;
 				$envvariables['NSUPDATE_SERVER'] = $nsupdatefileprefix;
 				$envvariables['NSUPDATE_KEY'] = $nsupdatefileprefix;
 				$envvariables['NSUPDATE_ZONE'] = $domain->NSUPDATE_ZONE;
+				if (strpos($domain->NSUPDATE_SERVER, ":") !== false)
+				{
```

Comment:
Please use ["K&R, BSD KNF variant style" formatting](https://docs.netgate.com/pfsense/en/latest/references/developer-style-guide.html#indent-style).

Reply:
Fixed in latest commit.